content_store_handler.go

```diff
@@ -52,20 +45,12 @@ func (cs *ContentStoreHandler) PutContentItem(w http.ResponseWriter, r *http.Req
 		return
 	}
 
-	if !cs.registerWithURLArbiter(path, contentStoreRequest.PublishingApp, w) {
+	if !cs.registerWithURLArbiter(contentStoreRequest.BasePath, contentStoreRequest.PublishingApp, w) {
```

Comment:
We don't want to take the path from the JSON.  In the case where the base_path in the JSON request body doesn't match the path in the request, we should use the path in the request (that's what content-store does at the moment).  

Alternatively, we could reject these requests as invalid.